tests/Rules/BannedNodesRuleTest.php

```diff
@@ -130,8 +130,6 @@ protected function assertNodeTriggersError(BannedNodesRule $rule, Node $node): v
         $errors = $rule->processNode($node, $this->scope);
         $this->assertCount(1, $errors);
         $error = $errors[0];
-        $this->assertInstanceOf(RuleError::class, $error);
```

Comment:
Why?

Reply:
PHPStan points this line because the `processNode` function returns an array of `RuleError` objects.

So the assertion seems to be useless.